pkg/restic/restic.go

```diff
@@ -138,7 +140,7 @@ func resticArgs(profile *param.Profile, repository, encryptionKey string) []stri
 	var cmd []string
 	switch profile.Location.Type {
 	case crv1alpha1.LocationTypeS3Compliant:
-		cmd = resticS3Args(profile, repository)
+		cmd, _ = resticS3Args(profile, repository)
```

Comment:
Alternatives:
1) Check error and log it.
2) Check error and propagate it to the caller methods (Need to change function signatures in this package).

Reply:
let's propagate.